css/scss/sections/_index.scss

```diff
@@ -0,0 +1,228 @@
+body {
+  background-color: #307367;
+  @include breakpoint(medium) {
+  background-color: #fff;
+  }
+}
+
+
+p{
+  font-weight: bold;
+color:#fff;
+  @include breakpoint(medium) {
+ color: set-color('grey-700');
+  }
+}
+
+.loginCon {
+ padding: 0;
+
+
+
+    .row {
+    padding:0;
+    }  
+}
+
+.left-log  {
+  padding: 0;
+  margin: 0;
+.bgimg {
+    background-image: url('#{$img-path}/cto-bg2.jpg');
+    height: 100%;
+    min-height: 100vh;
+    background-size: cover;
+}
+}
+
+.right-log {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around; 
+  width: 70%; 
+   @include breakpoint(medium) {
+  justify-content: space-around; 
+  width: 57%; 
+  }
+   @include breakpoint(large) {
+  justify-content: flex-end; 
+  width: 41%;     
+  }
+}
+
+.whiteLogo {
+  margin-top: 80px;
+  width:150px;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+
+.vertical-logo img {
+  margin-top: 80px;
+  width:100px; 
+  @include breakpoint(large) {
+  width:150px;       
+  }
+}
+
+.checkbox {
+  font-weight: normal;
+  margin-top: auto;
+  margin-bottom: auto;
+  @include breakpoint(medium) {
+ color: set-color('grey-700');
+     }
+}  
+
+.link {
+  font-weight: normal;
+  margin-top: auto;
+  margin-bottom: auto;
+   color: #fff;
+  @include breakpoint(medium) {
+ color: set-color('grey-700');
+     }
+ }
+.register {
+  .column-small-6{
+width: 27%;    
+padding: 0rem;
+  }
+}
+.button-white {
```

Comment:
Use the buttons we already have built. 
There is the buttons scss file, you can use any of those classes for the buttons we already have. We didn't had any for the facebook button, so that is ok that you created, but if it is a button that is global for the entire site it will be on the buttons.scss file.

Reply:
exactly... i got this one from that file...

Reply:
ohhh i think i understand now... so would i use for the small format .stroke-green (i'm guessing this is the one for the green background) and then changed to .button-white for large?

Reply:
The login button on large is green and not white. Use the existing classes, it is quite probable that you won't need to do anything on css with it, because the buttons classes are already fully functional and work on focus and hover states as well.

Reply:
fixed. I used the buttons white and green that were already built by did some slight modifications to them. This shouldn't override anything else on the app because nested the buttons so this wouldn't be a problem.